fx-src/index.js

```diff
@@ -15,18 +15,30 @@
 
 "use strict";
 
-const Cu = Components.utils;
+const { classes: Cc, interfaces: Ci, utils: Cu }  = Components;
 
 import BaseAdapter from "../src/adapters/base";
 import KintoBase from "../src/KintoBase";
 import FirefoxAdapter from "./FirefoxStorage";
 
 export default function loadKinto() {
   const { EventEmitter } = Cu.import("resource://devtools/shared/event-emitter.js", {});
+  const { generateUUID } = Cc["@mozilla.org/uuid-generator;1"].getService(Ci.nsIUUIDGenerator);
 
   Cu.import("resource://gre/modules/Timer.jsm");
   Cu.importGlobalProperties(['fetch']);
 
+  // Leverage Gecko service to generate UUIDs.
+  function makeIDSchema() {
+    const RE_UUID = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/i;
```

Comment:
We should probably export and reuse the one defined there https://github.com/Kinto/kinto.js/blob/master/src/utils.js#L5